include/api.php

```diff
@@ -77,40 +76,40 @@ function pll_default_language( $field = 'slug' ) {
 }
 
 /**
- * Among the post and its translations, returns the id of the post which is in the language represented by $slug
+ * Among the post and its translations, returns the id of the post which is in the language represented by $slug.
  *
  * @api
  * @since 0.5
  *
- * @param int    $post_id post id
- * @param string $slug    optional language code, defaults to current language
- * @return int|false|null post id of the translation if exists, false otherwise, null if the current language is not defined yet
+ * @param int    $post_id Post id.
+ * @param string $slug    Optional language code, defaults to the current language.
```

Comment:
`$slug` might get confused with the post's slug.

Reply:
This is in fact the language slug, used in the url, but you are right on the risk of confusion. I changed this.